src/lib/mongoBackend/dbFieldEncoding.cpp

```diff
@@ -41,8 +41,9 @@
 * Replace:
 *   . => =
 */
-std::string dbDotEncode(std::string s)
+std::string dbDotEncode(const std::string& _s)
 {
+  std::string s = _s; // replace cannot be used in const std::string&
   std::replace(s.begin(), s.end(), ESCAPE_1_DECODED, ESCAPE_1_ENCODED);
   return s;
 }
```

Comment:
Let's see, this is a function that modifies the parameter, so the parameter should be sent using its pointer, I mean, real pointer, not a C++ reference